src/DiceRoller/Dice.hpp

```diff
@@ -0,0 +1,63 @@
+//
+//  Dice.hpp
+//  DiceRoller
+//
+//  Created by Francesco Benzi on 2019-09-20.
+//  Copyright © 2019 Francesco Benzi. All rights reserved.
+//
+
+#ifndef Dice_hpp
+#define Dice_hpp
+
+#pragma once
+#include <vector>
+#include <stdio.h>
+
+class Dice
+{
+    
+public:
+    
+    Dice();
+    
+    /*
+     * diceRoller() rolls one to three dice, registers the new %
+     * and returns a sorted container which holds the
+     * values of the dice.
+     */
+    std::vector<int> diceRoller(int numDice);
+    
+    /*
+     * printRollPercentageArray() is a function that prints the current
+     * rolling % for all dice values (from 1 to 6).
+     */
+    
+    void printRollPercentageArray();
+    
+    /*
+     * getRollPercentage() is a function that gathers the %
+     * value for a specific dice value.
+     */
+    
+    void getRollPercentage(int roll);
```

Comment:
void and get doesn't fit well in a signature ;) 

This comment fits with my earlier one, i think this should return a int or whatever and let the callee deal with the cout. 

This fits with the "S" of the SOLID principles. A class should have one responsability, and Dice certainly should not deal with I/o.

Reply:
As required, the function now will return a _double_ value when invoked. 
The signature has obviously been changed in the _.h_ file as well.